1-js/08-prototypes/02-function-prototype/4-new-object-same-constructor/solution.md

```diff
@@ -33,17 +33,17 @@ let user2 = new user.constructor('Pete');
 alert( user2.name ); // undefined
 ```
 
-Why `user2.name` is `undefined`?
+Por que `user2.name` está `undefined`?
 
-Here's how `new user.constructor('Pete')` works:
+Abaixo está como o `new user.constructor('Pete')` funciona:
 
-1. First, it looks for `constructor` in `user`. Nothing.
-2. Then it follows the prototype chain. The prototype of `user` is `User.prototype`, and it also has no `constructor` (because we "forgot" to set it right!).
-3. Going further up the chain, `User.prototype` is a plain object, its prototype is the built-in `Object.prototype`. 
-4. Finally, for the built-in `Object.prototype`, there's a built-in `Object.prototype.constructor == Object`. So it is used.
+1. Primeiro, ele procura o `constructor` do `user`. Não o encontra.
+2. Depois, ele segue a cadeia de protótipos. O protótipo de `user` é `User.prototype`, e ele também não tem um `constructor` (porque nós nos "esquecemos" de configurá-lo!).
+3. Indo adiante na cadeia, `User.prototype` é um objeto vazio, cujo protótipo é o `Object.prototype` padrão.
+4. Por último, para o `Object.prototype` padrão existe um `Object.prototype.constructor == Object` padrão. Então, ele é usado.
 
-Finally, at the end, we have `let user2 = new Object('Pete')`. 
+No fim, nós temos `let user2 = new Object('Pete')`.
 
-Probably, that's not what we want. We'd like to create `new User`, not `new Object`. That's the outcome of the missing `constructor`.
+Provavelmente não é isso que nós queremos. Nós gostaríamos de criar um `new User`, não um `new Object`. Essa é a consequência da falta de um `constructor`.
 
-(Just in case you're curious, the `new Object(...)` call converts its argument to an object. That's a theoretical thing, in practice no one calls `new Object` with a value, and generally we don't use `new Object` to make objects at all).
\ No newline at end of file
+(Caso você esteja curioso, a chamada `new Object(...)` converte o seu argumento para um objeto. Isso é algo teórico, mas na prática ninguém faz a chamada `new Object` com um valor, e geralmente nós não usamos `new Object` para criar objetos).
```

Comment:
> Locuções entre vírgulas: portanto, por exemplo, na verdade [VERB_COMMA_CONJUNCTION](https://community.languagetool.org/rule/show/VERB_COMMA_CONJUNCTION?lang=pt)

- This phrase must be separated by commas, and must only be used at the beginning of a sentence for stylistic purposes. **Change suggestion**: `mas, na prática,`

<img width="674" alt="image" src="https://user-images.githubusercontent.com/35238611/177466351-f553130d-1aab-4829-879b-b6eeeb07d72f.png">